addons/turbowarp-player/style.css

```diff
@@ -26,19 +26,22 @@
   background-image: url(https://scratch.mit.edu/favicon.ico);
 }
 
-.sa-tw-iframe {
-  overflow: hidden;
-  box-sizing: border-box;
-
-  /* Same styles as .project-description */
-  border: 1px solid rgba(77, 151, 255, 0.1);
-  border-radius: 8px;
-  background-color: rgba(77, 151, 255, 0.1);
-
+.sa-tw-iframe-container {
   /* Same styles as .guiPlayer */
   display: inline-block;
   position: relative;
   z-index: 1;
   width: 482px;
   height: 406px;
+
+  /* Hide part of loading screen to make it the same size as Scratch's */
+  overflow: hidden;
+}
+
+.sa-tw-iframe {
+  width: calc(100% + 8px);
+  height: calc(100% + 10px);
+  margin-left: -4px;
+  margin-top: -3px;
```

Comment:
And can you add comments on where these magic numbers come from? Or is it by eye?

Reply:
I added the comments.